src/v3/src/transformer/button/transformReturnToAuthenticatorListButton.ts

```diff
@@ -33,13 +36,44 @@ export const transformReturnToAuthenticatorListButton: TransformStepFnWithOption
     return formbag;
   }
 
+  const onClick = (widgetContext?: IWidgetContext): unknown => {
+    if (typeof widgetContext === 'undefined') {
+      return;
+    }
+    const { setIdxTransaction } = widgetContext;
+    const availableSteps = transaction.availableSteps?.filter(
+      ({ name }) => name !== IDX_STEP.SELECT_AUTHENTICATOR_ENROLL,
```

Comment:
Curious if in general it's safe to assume the select-authenticator-enroll step will exist in the availableSteps array if this transformer got called. Similar question for select-authenticator-authenticate

Reply:
From what I have been told (and what I have seen), the object in NextStep should be equivalent to `availableSteps[0]`, the nextStep should be derived from the availableSteps. I have seen a bug (prior to introducing generic remediator) where some steps were missing or not in the correct order but I am pretty sure that is why GR was introduced (to resolve those types of issues).